tests/QAN/pages/qanDetailsFragment.js

```diff
@@ -24,10 +24,16 @@ module.exports = {
 
   async verifyAvqQueryCount(timeRangeInSec = 300) {
     const qpsvalue = await I.grabTextFrom(this.getMetricsCellLocator('Query Count', 2));
-    const queryCountDetail = await I.grabTextFrom(this.getMetricsCellLocator('Query Count', 3));
+    let queryCountDetail = await I.grabTextFrom(this.getMetricsCellLocator('Query Count', 3));
+
+    if (queryCountDetail.indexOf('k', 0) > -1) {
```

Comment:
https://www.w3schools.com/jsref/jsref_endswith.asp

you should use `string.endsWith()`

Reply:
ok.thanks